backend/app/views/spree/admin/orders/_adjustments.html.erb

```diff
@@ -3,8 +3,8 @@
     <legend><%= title %></legend>
     <table>
       <thead>
-        <th><%= Spree.t('name')%></th>
-        <th><%= Spree.t('amount')%></th>
+        <th><%= Spree::Adjustment.human_attribute_name(:name)%></th>
```

Comment:
Exactly. I didn't want to change the original appearance of the page. We can change that later on, as `label` would be the correct attribute.